modules/nupac.nu

```diff
@@ -11,11 +11,35 @@
 #? - package
 #? - management
 
+# If the enviroment variable exists
+def is-in-env [name: string] {
+    not ($env | get -i name | empty?)
+}
+
+# Specified value, env value or hardcoded value of the flag
+def get-flag-value [
+    flag: bool
+    env-var: string
+    ] {
+    if ($flag) {
+        true
+    } else if (is-in-env $env-var) {
+        $env | get $env-var
+    } else {
+        false
+    }
+}
+
 # Path where packages will be installed, can be changed to any other directory provided it is added to NU_LIB_DIRS variable
 def scripts-path [] {
-    $nu.config-path
-    |path dirname
-    |path join 'scripts'
+    if (is-in-env "NUPAC_DEFAULT_INSTALL_DIR") {
```

Comment:
Maybe change the variable name to NUPAC_DEFAULT_LIB_DIR since tha't what nushell itself calls them